### DISCUSSION TITLE
Cache busting a deployed site

### DISCUSSION THREAD

**Discussion by mikepianka** [thuliteio/doks#521] [Q&A] [answered]
I'm having trouble getting changes to show up properly in a site that was already deployed. It's deployed on Apache and I have caching turned off on index.html and all xml files. I don't do much other work with Hugo, but for my React apps this is usually all that's needed. In Chrome I am seeing partial changes until I manually clear the cache, then I see all the changes I made as expected. Is there anything I am supposed to be configuring specifically for the theme or Hugo to handle this? Thanks.

**Comment by h-enk** [comment, marked as answer]
> In Chrome I am seeing partial changes until I manually clear the cache, then I see all the changes I made as expected.

Chrome caches quite aggressively /that looks like standard Chrome behavior.

> Is there anything I am supposed to be configuring specifically for the theme or Hugo to handle this?

No, the default settings should do — see also [Configure File Caches](https://gohugo.io/getting-started/configuration/#configure-file-caches) — unless your adding dynamic elements (like e.g. React or reading from json files). For example, for getdoks.org I'm using these settings to display the most recent Doks version of `package.json`:

```toml
[caches]
  [caches.getjson]
    dir = ":cacheDir/:project"
    maxAge = "10s"
```

**Reply by mikepianka** [reply]
OK thanks 👍 . Just wanted to make sure I wasn't missing something.
